package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0-beta12",
   "description": "Premium and Open Source dashboard template with responsive and high quality UI.",
   "homepage": "https://tabler.io",
+  "type": "commonjs",
```

Comment:
I think this line should be removed (this source should use `import` instead of `require()`.